src/common/stores/emote-menu-view-store.js

```diff
@@ -169,6 +193,27 @@ class EmoteMenuViewStore extends SafeEventEmitter {
         emotes.getEmotesByCategories([EmoteCategories.FRANKERFACEZ_GLOBAL])
       ),
     ];
+
+    for (const [registeredCategoryId, registeredCategory] of Object.entries(registeredProviderCategories)) {
+      if (registeredCategory?.channelId != null && registeredCategory?.channelId !== currentChannel?.id) {
+        continue;
+      }
+      const provider = this.getRegisteredProvider(registeredCategory.providerId);
+      if (provider == null) {
+        continue;
+      }
+      const category = createCategory(
+        registeredCategoryId,
+        registeredCategory.providerId,
+        `${provider.displayName} ${registeredCategory.channelId == null ? 'Global' : 'Channel'}`,
```

Comment:
this doesn't support i18n, so that's a bit of a pickle. we prob need two strings

```
{providerName} Global
```

```
{providerName} Channel
```

Reply:
this hasn't been addressed, so marking unresolved.